src/styles/custom.css

```
@@ -36,13 +36,45 @@
 
 @media (max-width: 49.999rem) {
+  :root {
+    --sl-menu-button-size: 2.5rem;
+  }
+
   starlight-menu-button button[aria-controls='starlight__sidebar'] {
     --zs-mobile-header-height: calc(var(--sl-nav-height) - var(--sl-announcement-height));
     --zs-menu-button-offset: calc(
       (var(--zs-mobile-header-height) - var(--sl-menu-button-size)) / 2
     );
+    align-items: center;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)